### PR TITLE
Feature/mdr 329

### PR DESCRIPTION
**Pull request by mclaurentiu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,21 +78,21 @@ service | URL | info
 
 > Go to this link [localhost:4020/ui](http://localhost:4020/ui) to access the online definition of the API.
 
-![fingerprinter-api](docs/images/fingerprinter-api.png)
+![fingerprinter-api](./images/fingerprinter-api.png)
```
Comment (cosletmihai) [inline README.md:81]: the path doesn't work anymore and the images are not shown in the readme

**Review by cosletmihai — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (costezki) [reply README.md:81]: I assume we need to write `images/fingerprinter-api.png` instead of `./images/fingerprinter-api.png`;


Or something like that.

**Review by costezki — APPROVED** [review]
well done

File: README.md [hunk 2]
```diff
@@ -74,21 +78,21 @@ service | URL | info
 
 > Go to this link [localhost:4020/ui](http://localhost:4020/ui) to access the online definition of the API.
 
-![fingerprinter-api](docs/images/fingerprinter-api.png)
+![fingerprinter-api](images/fingerprinter-api.png)
```
Comment (costezki) [inline README.md:81]: oh! just realised that the readme will not be rendered properly. So perhaps we shall create the symlink in the opposite direction. 
`index.md -> ../readme.md`

**Review by costezki — APPROVED** [review]

**Pull request merged**